src/sources/lolalytics.js

```diff
@@ -0,0 +1,304 @@
+      blocks.push(block);
+      block = convertBuildSectionJsonToBlock(R.reverse(sortByPickRate(current_section.json)), `${T.t('most_freq', true)} ${T.t(current_section.title.split(' ').join('_'))}`);
+      blocks.push(block);
+    } else if (R.toLower(split_sort) === 'win') {
+      let block = convertBuildSectionJsonToBlock(R.reverse(sortByWinRate(current_section.json)), `${T.t(current_section.title.split(' ').join('_'))}`);
+      blocks.push(block);
+    } else {
+      let block = convertBuildSectionJsonToBlock(R.reverse(sortByPickRate(current_section.json)), `${T.t(current_section.title.split(' ').join('_'))}`);
+      blocks.push(block);
+    }
+  }
+
+  let riot_json = R.merge(R.clone(default_schema, true), {
+    champion: champion_name,
+    title: `Lolalytics ${champion_name} ${T.t(R.toLower(position), true)} ${store.get('lolalytics_ver')}`,
+    blocks: trinksCon(blocks, skills)
+  });
+
+  if (split_sort) {
+    if (R.toLower(split_sort) === 'win') {
+      riot_json.title += ' ' + T.t('highest_win', true);
+    } else {
+      riot_json.title += ' ' + T.t('most_freq', true);
+    }
+  }
+
+  if (store.get('settings').locksr) riot_json.map = 'SR';
+
+  return createChampionifyJson(riot_json, position, champion_name, split_sort);
+}
+
+function parseSkillsData(cheerio_data) {
+  let $c = cheerio_data;
+  let skill_sections = [];
+  $c('td.summarytitle').each(function(i, element) {
+    if ($c(this).text() === 'Abilities') {
+      let innerSkillsTable = $c(this).parent().find('table.summaryskillstable');
+      let rows = $c(innerSkillsTable).find('tr');
+
+      $c(rows).each(function(j, row) {
+        if (j === 0)
+          return; //  Skip the header row
+
+        let skill_order = $c(row).find('td.summaryskills > div.summaryspellkey').text().split('').join('.');
+        let win_rate = parseFloat($c(row).find('td.win').text());
+        let pick_rate = parseFloat($c(row).find('td.popularity').text());
+
+        let skill_section = {
+          skills: skill_order,
+          win: win_rate,
+          pick: pick_rate
+        };
+        skill_sections.push(skill_section);
+      }); //  end each row
+    } //  end if abilities section
+  }); //  end each td.summarytitle
+
+  var highest_win_func = R.sortBy(R.prop('win'));
+  var most_freq_func = R.sortBy(R.prop('pick'));
+
+  let highest_win_skills = R.reverse(highest_win_func(skill_sections))[0];
+  let most_freq_skills = R.reverse(most_freq_func(skill_sections))[0];
+
+  let shorthand_bool = store.get('settings').skillsformat;
+
+  let skills = {
+    most_freq: shorthand_bool ? shorthandSkills(most_freq_skills.skills.split('.')) : most_freq_skills.skills,
+    highest_win: shorthand_bool ? shorthandSkills(highest_win_skills.skills.split('.')) : highest_win_skills.skills
+  };
+
+  return skills;
+}
+
+function parseItemData(cheerio_data) {
+  let build_sections = [];
+  let $c = cheerio_data;
+
+  $c('td.summarytitle').each(function(i, element) {
```

Comment:
`$c('td.summarytitle').each(function(i, element) {`
I can get it to work using .map for this line instead of each.

Do you want it to be worked using map in the next few lines of that function where it's adding the name, pick, and win rate as well or can those stay as .each?